manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -110,6 +110,10 @@ public void init(Container container) throws Exception {
         LOG.info("\t- manager_config.json: " + managerConfigPath);
         LOG.info("\t- mapsettings.json: " + mapSettingsPath);
         LOG.info("\t- mapdata.mbtiles: " + mapTilesPath);
+        Path customMapTilesPath = getCustomMapTilesPath();
+        if (customMapTilesPath != null) {
+            LOG.info("\t- " + customMapTilesPath.getFileName() + ": " + customMapTilesPath);
+        }
```

Comment:
This logic shouldn't be needed, should just output mapTilesPath which should resolve to custom file if set (see file comment I have added).